tests/simulate/test_hoomd.py

```diff
@@ -0,0 +1,183 @@
+"""Unit tests for relentless.simulate.hoomd."""
+import tempfile
+import unittest
+
+import gsd.hoomd
+import hoomd
+import numpy as np
+
+import relentless
+
+class test_HOOMD(unittest.TestCase):
+    """Unit tests for relentless.HOOMD"""
+
+    def setUp(self):
+        self._tmp = tempfile.TemporaryDirectory()
+        self.directory = relentless.Directory(self._tmp.name)
+
+    #mock (NVT) ensemble and potential for testing
+    def ens_pot(self):
+        ens = relentless.Ensemble(T=100.0, V=relentless.Cube(L=10.0), N={'A':2,'B':3})
```

Comment:
If you are going to call `run` anywhere, I would suggest making T order 1 or else things will likely blow up.
```suggestion
        ens = relentless.Ensemble(T=2.0, V=relentless.Cube(L=10.0), N={'A':2,'B':3})
```